Resources/Private/Template/CopyElastic.sh.template

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env bash
 export NODE_TLS_REJECT_UNAUTHORIZED=0
 
-ssh {remoteInstance.sshOptions->f:format.raw()} -C -f -N -L 9210:{remoteConfiguration.elasticsearchHost}:{remoteConfiguration.elasticsearchPort} {remoteInstance.user}@{remoteInstance.host}
+ssh {remoteInstance.sshOptions->f:format.raw()} -C -f -N -L 127.0.0.1:9210:{remoteConfiguration.elasticsearchHost}:{remoteConfiguration.elasticsearchPort} {remoteInstance.user}@{remoteInstance.host}
```

Comment:
We only need one loopback interface and `::1` might not be available (e.g. in Docker).